tests/pipelines/multi_asic_201911.yml

```diff
@@ -0,0 +1,137 @@
+# Starter pipeline
+
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+
+# Add steps that build, run tests, deploy, and more:
+
+# https://aka.ms/yaml
+
+pr:
+- master
+
+name: $(TeamProject)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+parameters:
+- name: dut
+  type: string
+  default: vlab-08
+- name: tbname
+  type: string
+  default: vms-kvm-four-asic-t1-lag
+- name: ptf_name
+  type: string
+  default: ptf_test
+- name: tbtype
+  type: string
+  default: multi-asic-t1-lag
+
+resources:
+  repositories:
+  - repository: sonic-mgmt
+    type: github
+    name: Azure/sonic-mgmt
+    endpoint: build
+
+stages:
+- stage: Test
+  pool: sonictest
+  variables:
+  - name: inventory
+    value: veos_vtb
+  - name: testbed_file
+    value: vtestbed.csv
+  - name: image_file_path
+    value: https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-multiasic-vs-image-201911/lastSuccessfulBuild/artifact/target/sonic-vs.img.gz 
```

Comment:
jenkins is going to be deprecated. please download the artifacts from azp to avoid disruption.

Reply:
Sure, will add azp job to build multi-asic vs image and modify script to use the right image.